src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Herentals.cs

```diff
@@ -0,0 +1,33 @@
+namespace Basisregisters.IntegrationDb.NationalRegistry.Model.HouseNumberBoxNumberImplementations.Municipalities
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class Herentals : MunicipalityHouseNumberBoxNumbersBase
+    {
+        public Herentals(string nisCode, string sourceHouseNumber, NationalRegistryIndex index) : base(nisCode, sourceHouseNumber, index)
+        { }
+
+        public override bool IsMatch() =>
+            NisCode == "13011" &&
+            (
+                IndexSourceValue.Length == 4 && IsLetter(IndexSourceValue[0]) && IndexSourceValue[1..] == "000"
```

Comment:
IndexSourceValue is always 4 characters long.
![image](https://github.com/Informatievlaanderen/basisregisters-integration-db/assets/18366653/a83320ed-90cc-4f6e-82c4-bed3a6a3b928)